### DISCUSSION TITLE
How do we get people to socialize a bit more?

### DISCUSSION THREAD

**Discussion by TomCools** [BeJUG/bejug.github.io#19] [Ideas]
Let's discuss!

- Namelabels?
- Assign tables? :-)

**Comment by denmette** [comment]
I want to propose some topics that are already discussed but our more opinionated. Those can be put on tables around the location where people can debate that topic.

- Spaces vs. Tabs
- Dark vs. Light theme

Connecting with a name badge is more accessible, but that should be fine.
